ulexecve.py

```diff
@@ -875,7 +875,7 @@ def mprotect(self, addr, length, prot):
 
     def munmap(self, addr, length):
         """
-        b8 0b 00 00 00       	mov    $0xb,%eax
+        b8 5b 00 00 00       	mov    $0x5b,%eax
```

Comment:
Also, curious: the asm arguments look like they are all backwards, is that some sort of convention?
(I would expect to read `mov %eax,$0x5b` as in `mov dst,src`)

Reply:
I've been told att vs intel notation.